users/templates/base.html

```diff
@@ -24,9 +24,13 @@
         <tr><br /></tr>
         <tr>
           <td class="padding"><a href="{% url 'home' %}">{% trans 'Home' %}</a></td>
-          <td class="padding"><a href="{% url 'custom-user-update' user.id %}">Account</a></td>
+          <td class="padding"><a href="{% url 'custom-user-update' user.id %}">{% trans 'Account' %}</a></td>
             {% if user.user_type == "ADMIN" %}
           <td class="padding"><a href="{% url 'custom-users-list' %}">{% trans 'Employees' %}</a></td>
+          <td class="padding"><a href="{% url 'custom-projects-list' %}">{% trans 'Projects list' %}</a></td>
+            {% endif %}
+            {% if user.user_type == "MANAGER" %}
```

Comment:
This is a good candidate for custom template tag. This way we could use `"MANAGER"` value directly from Python enum, in case if its value would change in the future.

Reply:
https://memegenerator.net/img/instances/80356275/consider-it-done.jpg